main/acle.md

```diff
@@ -9134,130 +9134,130 @@ Replacing `_hor` with `_ver` gives the associated vertical forms.
 
 ``` c
```

Comment:
Does svldr also need a `_m` suffix, given that the vector is loaded into the ZA array?

Reply:
Sorry, I just realise that question makes no sense, the `_m` suffix is there in the context of predication such that only part of the _tile_ is updated (not that part of the _ZA array as a whole_ is updated).